src/scripts.js

```diff
@@ -154,6 +261,7 @@ function makeSleepQualityHTML(id, sleepInfo, userStorage, method) {
 }
 
 function addActivityInfo(id, activityInfo, dateString, userStorage, laterDateString, user, winnerId) {
+  // addDailyActivityInfo(id, activityInfo, dateString, userStorage);
```

Comment:
Do we need this commented out?

Reply:
Oh! This should be deleted! I apologize.

Reply:
Fixed!